src/pages/index.tsx

```diff
@@ -24,6 +24,17 @@ const Index: React.FC<IndexProps> = ({ fetchedData }) => {
 
   const { user } = fetchedData.data.provider;
 
+  let emailCount = 0;
+  let phoneCount = 0;
+
+  user.identities.map(({ type }) => {
+    if (type === IdentityTypes.EMAIL.toLowerCase()) {
+      emailCount++;
+    } else if (type === IdentityTypes.PHONE.toLowerCase()) {
+      phoneCount++;
+    }
+  });
```

Comment:
Also, `.map()` returns an array, which is not used here. If we would go for this implementation, `forEach()` would have been more efficient (as it returns nothing).

Reply:
OK fixed 79a61d5